src/messaging/ReliableMessageProtocolConfig.h

```diff
@@ -54,7 +54,7 @@ namespace Messaging {
  *
  */
 #ifndef CHIP_CONFIG_MRP_DEFAULT_ACTIVE_RETRY_INTERVAL
-#define CHIP_CONFIG_MRP_DEFAULT_ACTIVE_RETRY_INTERVAL (300)
+#define CHIP_CONFIG_MRP_DEFAULT_ACTIVE_RETRY_INTERVAL (2000)
```

Comment:
This goes against the default definition in the spec (section 4.15.8). Please overwrite for the specific platform only.

Reply:
Done